src/alchemlyb/tests/test_visualisation.py

```diff
@@ -218,6 +218,47 @@ def test_plot_convergence_final_nan():
     plt.close(ax.figure)
 
 
+def test_plot_block_average(gmx_benzene_Coulomb_u_nk):
```

Comment:
We might want to refactor this in to a class Instead of doing a couple test in a single function.

Reply:
I'm following the precedent of all the other tests in this file.